README.md

```diff
@@ -2,6 +2,34 @@
 
 This repository contains GCP connectors for Apache Flink.
```

Comment:
this isn't really true. this repo doesnt actually house any connectors (right now). it just houses examples for how to integrate GMF and OSS flink with GCP

Reply:
Removed this